subsys/debug/Kconfig

```diff
@@ -3,6 +3,8 @@
 # Copyright (c) 2015 Wind River Systems, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
+DT_CHOSEN_Z_CONSOLE := zephyr,console
```

Comment:
Does this cause inclusion of console stuff? (I am unaware, somebody else comment). 
If so, it should be made dependent on this tracing, as I can imagine not everybody forcibly wants console or device tree.

Reply:
I just use it to get uart name which is dedicated to tracing.
 
config TRACING_BACKEND_UART_NAME
        string "Device Name of UART Device for UART backend"
        default "$(dt_chosen_label,$(DT_CHOSEN_Z_CONSOLE))" if HAS_DTS
        default "UART_0"
        depends on TRACING_BACKEND_UART
        help
          This option specifies the name of UART device to be used for
          tracing backend.